ztp/gitops-subscriptions/argocd/README.md

```diff
@@ -123,6 +122,11 @@ In order to deploy the OpenShift GitOps operator v1.12 you may apply the provide
     oc apply -k out/argocd/deployment
 ```
 
+7. Optionally: If configuring an existing hub cluster (i.e skipped step 6, pipeline configuration), starting with ACM 2.10, the following patch is needed to allow ACM to update `vendor` and `cloud` labels in ManagedCluster CRs. This enables observability without user intervention.
+```
+    $ oc patch applications.argoproj.io clusters -n openshift-gitops --type='json' --patch-file out/argocd/deployment/allow-acm-to-override-vendor-and-cloud-value.json
```

Comment:
Does it mean that patching this json file will add those labels under the ManagedCLusterCR for the existing deployment?

Reply:
Yeah essentially! So the labels will be there in the ManagedCluster CR right away. But with this patch...argocd will allow "vendor: auto-detect" to update to appropriate value (e.g "vendor: OpenShift"). 

Without this patch argocd will not sync since by default it always tries to match all the content that's in git with the live cluster. So ACM tries to update and Argocd reverts